tests/helpers/assert-async-throws.js

```diff
@@ -0,0 +1,17 @@
+import QUnit from 'qunit';
+
+QUnit.extend(QUnit.assert, {
+  asyncThrows: async function(func, regex, message = '') {
```

Comment:
Umm, why?

Is there an issue with `assert.rejects`? 

https://api.qunitjs.com/assert/rejects